src/listing.ts

```diff
@@ -156,9 +177,23 @@ class DirListing extends Widget {
    */
   static createNode(): HTMLElement {
     let node = document.createElement('div');
-    let content = document.createElement('ul') as HTMLElement;
-    content.className = LIST_AREA_CLASS;
-    node.appendChild(content);
+    let header = document.createElement('div');
+    let body = document.createElement('div');
+    let contents = document.createElement('ul');
+    header.className = LIST_HEADER_CLASS;
+    body.className = LIST_CONTAINER_CLASS;
+    contents.className = LIST_AREA_CLASS;
+    node.appendChild(header);
+    node.appendChild(body);
+    body.appendChild(contents);
+    let files = document.createElement('span');
+    let modified = document.createElement('span');
+    files.textContent = 'Files';
+    files.className = HEADER_FILE_CLASS;
+    modified.textContent = 'Last Modified';
```

Comment:
Same question: is this language going to be passed in / configurable at some point?